scripts/map.js

```diff
@@ -468,7 +471,35 @@ var loadFederalState = function () {
 			},
 		})
 
-		overlays['fs'].addTo(map)
+		overlays['federal'].addTo(map)
+	})
+}
+
+//* state land overlay
+var loadState = function () {
+	$.getJSON('./data/state-land.min.geojson', function (geojson) {
+		var stripes = new L.StripePattern({
+			height: 2,
+			width: 2,
+			weight: 1,
+			spaceWeight: 1,
+			angle: 30,
+			color: '#5cc649',
```

Comment:
```suggestion
			color: '#b349c6',
```
![Screenshot 2022-12-17 at 6 28 52 PM](https://user-images.githubusercontent.com/15609358/208281530-0474a3ae-4973-4946-ba87-14c7eb2ae6de.png)

Thinking about it some more, we should probably color these differently. Randomly picking a color based off what a random website told me to

https://colorpicker.me/#b349c6

Reply:
Suggestion committed ➡️  [1df5d29](https://github.com/CodeforHawaii/Hawaii-Zoning-Atlas/pull/47/commits/1df5d294e45ea3139179443f68cb4981fb05ae16)